dataline-workers/src/main/java/io/dataline/workers/singer/SingerTapFactory.java

```diff
@@ -135,8 +152,7 @@ public Runnable getCloseFunction() {
     StandardDiscoverSchemaInput discoveryInput = new StandardDiscoverSchemaInput();
     discoveryInput.setConnectionConfiguration(
         input.getSourceConnectionImplementation().getConfiguration());
-    Path scopedWorkspace = workspaceRoot.resolve("discovery");
-    return new SingerDiscoverSchemaWorker(imageName, pbf)
-        .runInternal(discoveryInput, scopedWorkspace);
+    Path scopedWorkspace = workspaceRoot.resolve(DISCOVERY_DIR);
```

Comment:
can you rename `workspaceRoot` to `jobRoot`? (I missed that spot. the idea is workspace is the root dir and job root is the scoped dir)